lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorManager.java

```diff
@@ -276,7 +278,16 @@ public void onNext(final Alarm alarm) {
    */
   public boolean isClosed() {
     return this.messageDispatcher.isEmpty() &&
-           this.stateManager.isDoneOrFailedOrKilled();
+        (this.stateManager.isDoneOrFailedOrKilled());
```

Comment:
A minor syntactic note: the extra brackets around `this.stateManager.isDoneOrFailedOrKilled` are not necessary.